models/playlist.ts

```diff
@@ -17,4 +17,41 @@ module.exports.savePlaylist = function (name, songs, user_id, type, startdatetim
             return callback(err, result);
         }
     })
+};
+module.exports.saveOrUpdatePlaylist = function (name, songs, user_id, type, startdatetime, enddatetime, callback) {
```

Comment:
In principe kan je hier 3 functies uit halen, namelijk: 

 update Playlist (verander een bestaande, is in principe niet nodig maar wel mooier), countPlaylists, deletePlaylists. Dan kan jij (of kunnen anderen) ze hergebruiken. Plus is de code leesbaarder!

Ook zou een savePlaylist (is er al, staat er boven :)) toepasselijk zijn.